addons/product/views/product_packaging_views.xml

```diff
@@ -75,7 +75,6 @@
     <record model="ir.actions.act_window" id="action_packaging_view">
         <field name="name">Product Packagings</field>
         <field name="res_model">product.packaging</field>
-        <field name="domain">[('product_id', '!=', False)]</field>
```

Comment:
Force an empty value, or else the domain won't be changed on update.
```diff
-        <field name="domain">[('product_id', '!=', False)]</field>
+        <field name="domain">[]</field>
```

Reply:
@KangOl Maybe it's more clean to remove the domain in the migration script ? And maybe this should be more consistent too because if I apply your suggestion, we don't know if this empty domain will stay in the future (e.g.: somebody could remove it, thinking of cleaning, in another commit)

Reply:
No, it's more clear. You better be explicit. Here you clearly want an empty domain.